<launch>
	<node pkg="nccr_manet" name="xbee_manager" type="xbee_manager">
	      <param name="dev" value="/dev/ttyUSB0"/>
	</node>
	<node pkg="nccr_manet" name="xbee_disaggregator" type="xbee_disaggregator">
	</node>
	<node pkg="nccr_manet" name="debug_reporter" type="debug_reporter">
	</node>

	<node pkg="nccr_manet" name="flow_reporter" type="flow_reporter">
	</node>

	<node pkg="nccr_manet" name="lcm_flow_forwarder" type="lcm_flow_forwarder">
	      <param name="address_book" value="/root/addressbook"/>
	      <remap from="out_flows" to="flows"/>
	</node>

	<node pkg="nccr_manet" name="lcm_route_forwarder" type="lcm_route_forwarder">
	      <param name="lcm_topic" value="RNP2"/>
	</node>


</launch>
